specs/decisionmatrix/yaml2csv.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import yaml
+
+
+def yaml2csv(fname):
+    ''' Converts drive-YAML syntax to CSV '''
```

Comment:
just curious, what is the case that requires this conversion?

Reply:
This should help with fixing/review/add the drive params in JAML-files:
* this is much easier done via Excel tables, than raw YAML
* so, one can convert params to CSV, edit in Excel/Google-Sheets, and convert back to YAML